msteams-platform/toolkit/provision VS.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Use Teams Toolkit to provision cloud resources using Teams Toolkit for Visual Studio
+author: surbhigupta
+description: In this module, learn how to provision cloud resources using Teams Toolkit. Also to create resources and customize resource provision in Visual Studio
+ms.author: shenwe
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 11/29/2021
+---
+
+# Provision cloud resources using Visual Studio
+
+TeamsFx integrates with Azure and Microsoft 365 cloud that allows you to place your application in Azure with a single command. TeamsFx integrates with Azure Resource Manager that enables you to provision Azure resources. For the code approach, your application needs the cloud resources.
+
+## Prerequisites
+
+Here's a list of tools you need for provisioning your cloud resources:
+
```

Comment:
Visual Studio 2022 v17.3 is a prerequisite too.

Reply:
This comment has been implemented.